### PR TITLE
Add a unit test section to README file

### PR DESCRIPTION
**Pull request by solker**
unit_test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,24 @@ adding -d top the command:
...
-
-
-
-
+## Unit testing
+   
+For testing use Behat with PhantomJS and SeleniumHQ. To install this software do next steps:
+
+#### Install and run PhantomJS.
+Use next [instruction](https://github.com/Gizra/KnowledgeBase/wiki/Behat-phantomJs-install)
```
Comment (nader77) [inline README.md]: 'next' => 'the following'

File: README.md [hunk 2]
```diff
@@ -103,7 +103,24 @@ adding -d top the command:
...
+
+#### Install and run SeleniumHQ
+
+Download Selenium Server from next [link](http://docs.seleniumhq.org/download/).
+And run by next command:
```
Comment (nader77) [inline README.md]: 'by next' => 'with the following'

**Comment by solker** [comment]
@nader77 fixed.

**Comment by nader77** [comment]
@HelenaEksler Check the update on README, If it's ok assign to Peter.

**Comment by HelenaEksler** [comment]
@nader77 @solker I think we should add to the instruction creating behat.local.yml file.

**Comment by solker** [comment]
@nader77 @HelenaEksler fixed.

**Comment by HelenaEksler** [comment]
@solker Looks fine. I'll assign it to Peter.

**Pull request merged**